src/scripts/i18n/inline_and_diff.py

```
@@ -0,0 +1,201 @@
+            val = get_translation(key, translations)
+            if isinstance(val, str):
+                return val
+        return m.group(0)
+    def _html(m):
+        key = m.group(2)
+        if key.startswith(key_prefix):
+            val = get_translation(key, translations)
+            if isinstance(val, str):
+                return f"'{val}'"
+        return m.group(0)
+    def _ts(m):
```

Ruff (0.11.9)

Missing return type annotation for private function `_ts`

(ANN202)